hw/top_earlgrey/data/ip/chip_edn_testplan.hjson

```diff
@@ -130,16 +130,27 @@
             - Disable the entropy complex.
             - Enable ENTROPY_SRC in FIPS mode.
             - Enable CSRNG.
-            - Enable EDN0 in auto request mode.
-            - Enable EDN1 in auto request mode with the following configuration:
+            - Enable EDN1 in auto request mode.
+            - Enable EDN0 in auto request mode with the following configuration:
               - Set flag0 in all application command headers to kMultiBitBool4True.
                 This causes CSRNG to ignore the seed provided by ENTROPY_SRC and exclusively use the optional additional data provided by EDN.
               - Set the command length or amount of optional additional data for the Instantiate and Reseed commands to 12 words.
+              - Set the generate length parameter for the Generate command to 4 blocks.
+              - Provide the correct seed material for the Instantiate and Reseed commands.
+              - For an example, refer to the `chip_sw_csrng_kat_test`.
+            - Verify that EDN0 delivers the expected entropy bits using the rv_core_ibex.RND_DATA and rv_core_ibex.RND_STATUS registers.
+            - Disable the entropy complex.
+            - Enable ENTROPY_SRC in FIPS mode.
+            - Enable CSRNG.
+            - Enable EDN1 in auto request mode.
+            - Enable EDN0 in auto request mode with the following configuration:
+              - Set flag0 in all application command headers to kMultiBitBool4True.
```

Comment:
I think that "flag0" is a term from the CSRNG world? To make that a bit easier to pick up, maybe it's best to say that explicitly with something like ".. command headers sent to the CSRNG to kMultiBitBool4True" ?

Reply:
Thanks, can do!